nextflow_schema.json

```
@@ -53,7 +53,8 @@
                 "force_sratools_download": {
                     "type": "boolean",
                     "fa_icon": "fas fa-tools",
-                    "description": "Force download FastQ files via sra-tools instead of via the ENA FTP."
+                    "description": "Force download FastQ files via sra-tools instead of via the ENA FTP.",
```

Don't know if there is an overall preference in nf-core for FASTQ vs FastQ and FASTA vs FastA but I think all caps is the more common one?

```suggestion
                    "description": "Force download FASTQ files via sra-tools instead of via the ENA FTP.",
```